app/routes.py

```diff
@@ -1,2 +1,261 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import os
+import json, requests
+from dotenv import load_dotenv
 
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+load_dotenv()
+
+def post_message_to_slack(text):
+    SLACK_TOKEN = os.environ.get('SLACKBOT_TOKEN')
+    slack_path = "https://slack.com/api/chat.postMessage"
+    query_params ={
+        'channel': 'task-notifications',
+        'text': text  
+    }
+    headers = {'Authorization': f"Bearer {SLACK_TOKEN}"}
+    requests.post(slack_path, params=query_params, headers=headers)
+    
+@goals_bp.route("", methods=["GET", "POST"])
+def handle_goals():
+    if request.method == "GET":
+        goals = Goal.query.all()
+        goal_response = []
+        for goal in goals:
+            goal_response.append({
+                "title": goal.title,
+                "id": goal.goal_id
+            })
+        return jsonify(goal_response), 200
+    
+    elif request.method == "POST":
+        request_body = request.get_json()
+        title = request_body.get("title")
+        if not title:
+            return jsonify({"details": "Invalid data"}), 400
+        
+        new_goal = Goal(title=request_body["title"])
+        db.session.add(new_goal)
+        db.session.commit()
+        
+        commited_goal = {"goal":
+            {"id": new_goal.goal_id,
+            "title": new_goal.title
+            }}
+        
+    return (commited_goal), 201
+
+@goals_bp.route("/<goal_id>", methods=["GET", "PUT", "DELETE"])
+def handle_goal(goal_id):
+    goal = Goal.query.get_or_404(goal_id)
+
+    if request.method == "GET":
+        selected_goal = {"goal":
+            {"id": goal.goal_id,
+            "title": goal.title
+            }
+        }
+        return selected_goal
```

Comment:
just to be consistent I would add 

```suggestion
        return jsonify(selected_goal), 200
```